### PR TITLE
More DevDoc updates

### PR DESCRIPTION
**Pull request by Kredcool**

### CODE REVIEW

File: docs/checklist/event.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+The Event Checklist Subsystem
```
Comment (KenwoodFox) [inline docs/checklist/event.rst]: What lol? XD

File: docs/developing/developing.rst [hunk 2]
```diff
@@ -3,12 +3,6 @@ Developing
 
 .. include:: ../resources/resources.rst
 
-.. figure:: resources/AProgrammer.jpg
-  :width: 80%
-  :alt: Figure of a programmer
-
-  A programmer in his natural habitat.
-
```
Comment (KenwoodFox) [inline docs/developing/developing.rst:11]: :o

**Review by KenwoodFox — CHANGES_REQUESTED** [review]

File: docs/introduction/introduction.rst [hunk 3]
```diff
@@ -9,9 +9,13 @@ Introduction
...
-.. This should will be centered ..
-
-Welcome and thanks for reading
+.. This centers the Welcome and Thanks for Reading text ..
+.. yes I know this is stupid ..
+.. don't touch it ..
+.. figure:: resources/ThisIsStupid.png
+  :width: 300
+  :alt: Alternative text
+  :align: center
```
Comment (KenwoodFox) [inline docs/introduction/introduction.rst:18]: Thats cheating!

Comment (Kredcool) [reply docs/introduction/introduction.rst:18]: if you want a different solution please be my guest

Comment (KenwoodFox) [reply docs/introduction/introduction.rst:18]: I like it, please fix `docs/checklist/event.rst` tho

**Review by KhanSimeoni — APPROVED** [review]
whats with this code? looks kinda sus ngl

**Review by KenwoodFox — APPROVED** [review]
I love the placeholder image <3

**Pull request merged**